app/components/documents/document-badge-list.hbs

```diff
@@ -3,11 +3,20 @@
 as |documents|>
   <div class="grid-container">
     {{#each documents as |doc|}}
-      <Documents::DocumentBadge
-        @piece={{doc}}
-        @isHighlighted={{includes doc @highlightedPieces}}
-        @isClickable={{true}}
-      />
+      <li class="vlc-document-list__item">
+        <span>
+          <a
+            href={{doc.viewDocumentURL}} target="_blank" rel="noreferrer noopener"
+            data-test-document-badge-link
+          >
+            <Auk::Icon @name={{if (includes doc @highlightedPieces) "document-added" "document"}} />
+            {{doc.name}}
+          </a>
+          <AccessLevelPill
+            @accessLevel={{await doc.accessLevel}}
+          />
```

Comment:
Not  sure, but it seems to me on the usertest branch that the access-level is only shown for documents with level 'Intern secretarie' on the agenda overview page